contracts/governance/src/ZeroExProtocolGovernor.sol

```diff
@@ -0,0 +1,145 @@
+
+contract ZeroExProtocolGovernor is
+    Governor,
+    GovernorSettings,
+    GovernorCountingSimple,
+    GovernorVotes,
+    GovernorTimelockControl
+{
+    address public securityCouncil;
+
+    constructor(
+        IVotes _token,
+        ZeroExTimelock _timelock,
+        address _securityCouncil
+    )
+        Governor("ZeroExProtocolGovernor")
+        GovernorSettings(14400 /* 2 days */, 50400 /* 7 days */, 1000000e18)
+        GovernorVotes(_token)
+        GovernorTimelockControl(TimelockController(payable(_timelock)))
+    {
+        securityCouncil = _securityCouncil;
+    }
+
+    function quorum(uint256 blockNumber) public pure override returns (uint256) {
+        return 10000000e18;
+    }
+
+    // The following functions are overrides required by Solidity.
+
+    function votingDelay() public view override(IGovernor, GovernorSettings) returns (uint256) {
+        return super.votingDelay();
+    }
+
+    function votingPeriod() public view override(IGovernor, GovernorSettings) returns (uint256) {
+        return super.votingPeriod();
+    }
+
+    function state(uint256 proposalId) public view override(Governor, GovernorTimelockControl) returns (ProposalState) {
+        return super.state(proposalId);
+    }
+
+    function propose(
+        address[] memory targets,
+        uint256[] memory values,
+        bytes[] memory calldatas,
+        string memory description
+    ) public override(Governor, IGovernor) returns (uint256) {
+        return super.propose(targets, values, calldatas, description);
+    }
+
+    function proposalThreshold() public view override(Governor, GovernorSettings) returns (uint256) {
+        return super.proposalThreshold();
+    }
+
+    function cancel(
+        address[] memory targets,
+        uint256[] memory values,
+        bytes[] memory calldatas,
+        bytes32 descriptionHash
+    ) public {
+        require(msg.sender == securityCouncil, "ZeroExProtocolGovernor: Only security council allowed");
+        _cancel(targets, values, calldatas, descriptionHash);
+
+        // Eject security council
+        securityCouncil = address(0);
+    }
+
+    // Like the GovernorTimelockControl.queue function but without the proposal checks,
+    // (as there's effectively no proposal).
+    // And also using a delay of 0 as opposed to the minimum delay of the timelock
+    function executeRollback(
+        address[] memory targets,
+        uint256[] memory values,
+        bytes[] memory calldatas,
+        bytes32 descriptionHash
+    ) public {
+        require(msg.sender == securityCouncil, "ZeroExProtocolGovernor: Only security council allowed");
+
+        // Execute the batch of rollbacks via the timelock controller
+        ZeroExTimelock timelockController = ZeroExTimelock(payable(timelock()));
+        timelockController.executeRollbackBatch(targets, values, calldatas, 0, descriptionHash);
+
+        // Eject security council
+        securityCouncil = address(0);
```

Comment:
Emit an event? Same with all the other places `securityCouncil` gets set

Reply:
Added `SecurityCouncilAssigned` and `SecurityCouncilEjected` events here
https://github.com/0xProject/protocol/pull/641/commits/b17c03d3e03c0229e57beced2893a3eeef468f42